### ISSUE TITLE
Add TinyMCE Wysiwyg Field

### ISSUE THREAD

**Issue by JedWatson** [keystonejs/keystone-5#928]
Separate to #616, I want to add a WYSIWYG html field for the Admin UI.

It'll be useful while the content field is under development, and longer term helpful for transitioning from Keystone 4 or any other CMS that stores content in HTML.

The initial version should be a basic implementation of [TinyMCE 5](https://www.tiny.cloud/) and we should skin it a bit to fit in with the Admin UI style like we did in Keystone 4.

**Comment by gautamsi** [comment]
I hope you remember not to bundle the TinyMCE files like in Keystone v4 :) (which I got rid of recently)

**Comment by gautamsi** [comment]
How about a provider/adapter system with this field type? I am sure people would want to use something like [Trumbowyg](https://alex-d.github.io/Trumbowyg/).

if there is adapter/provider system, dev can add their own editor without creating whole new FieldType

**Comment by JedWatson** [comment]
I left a comment on #929 re: supporting other editors.

tl;dr:

Really all this package is doing is integrating and configuring TinyMCE. Field types already work like what you're describing, and I think adding other editors as separate field packages is already the right abstraction for this.

> I hope you remember not to bundle the TinyMCE files like in Keystone v4 :) (which I got rid of recently)

💯 I think you'll like what I've done!
